doc/schema.md

```diff
@@ -564,7 +564,7 @@ A table containing Plutus Datum, found in witnesses or inlined in outputs
 |-|-|-|
 | `id` | integer (64) |  |
 | `hash` | hash32type | The Hash of the Datum |
-| `tx_id` | integer (64) | The Tx table index for the transaction where this script first became available. |
```

Comment:
Indeed this should say `datum` instead of `script`. However I'm not sure if `was first used` is correct. At least in my mind a datum is used when it runs inside a script. But an inline datum is not used when it becomes available on chain.

Reply:
That makes sense (to my limited understanding - still very much learning!).

Maybe `where this datum was first observed` (or "mentioned", or "seen") then?